src/js/objectFitPolyfill.js

```diff
@@ -0,0 +1,281 @@
+/* ----------------------------------------
```

Comment:
We should probably import this polyfill as an actual npm devDependency instead of manually adding it to /src/.